src/client/user/components/UserLinkTable/UrlTable/EnhancedTableBody/index.tsx

```diff
@@ -129,6 +129,7 @@ const useStyles = makeStyles((theme) => {
     },
     clicksCellTooltipContent: {
       display: 'inline-block',
+      float: 'right',
```

Comment:
wondering why we're going with right-aligning the cells here? seems to add a lot of space between the last two columns

Reply:
To be honest, it was for ease of alignment with the status bar close button, so that the justification on the right from the close button to the end can be matched with the justification on the click count to the end. Had some issues with alignment otherwise

Will check with Feli if this is acceptable!